carmaelement_mod.F90

```diff
@@ -261,7 +266,7 @@ subroutine CARMAELEMENT_Print(carma, ielement, rc)
       write(carma%f_LUNOPRT,*) "    icomposition  : ", icomposition
       write(carma%f_LUNOPRT,*) "    isolute       : ", isolute
```

Comment:
Could/Shoud add kappa to the print.

Reply:
okay

Reply:
is done

Reply:
@fvitt needs to actually make a local variable...